source/administration/compliance-export.rst

```diff
@@ -32,6 +32,8 @@ Actiance Vantage Integration
 
 If you have chosen your file format to be Actiance XML, you can set up an integration with Actiance Vantage.
 
+Channel names will now be exported in the Actiance XML file with the channel type prepended.
```

Comment:
This document (compliance-export.rst) needs a few updates:
 - throughout, we say only Actiance is supported, but now both Actiance and Global Relay are supported (see introduction section, and step 3 of the set up guide)
 - there are additional settings for Global Relay to configure, which are not mentioned
 - there is no mention of how to configure it for Global Relay

As an additional note, we may need to revise the flow of the doc: right now it's Introduction > Set Up Guide > Actiance > Troubleshooting, but we'll want to support the flow for Global Relay as well

Reply:
@jasonblais Made some initial edits, but can you help with ` additional settings for Global Relay to configure` - I can't find them from System Console and I also have hard times filling in config settings doc for Global Relay settings.

Reply:
@amyblais Once you've chosen Global Relay in the compliance exports, you'll find the config settings

![image](https://user-images.githubusercontent.com/13119842/38517071-c9f6f978-3c06-11e8-951e-b2c2a7f0aed4.png)

Reply:
@jasonblais I tried earlier (on pre-release with my admin account and on ci-linux with bot@mattermost account) and the drop-down doesn't work - what account should I use to access?